rules.html

```diff
@@ -0,0 +1,171 @@
+
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+			rel="stylesheet"
+		/>
+
+		<link rel="stylesheet" href="static/css/style.css" />
+
+		<link
+			rel="shortcut icon"
+			href="static/img/favicon.ico"
+			type="image/x-icon"
+		/>
+
+		<title>OTC Talks Rules</title>
+	</head>
+
+	<body>
+		<header>
+			<nav>
+				<div class="burger-menu">
+					<div class="line-1"></div>
+					<div class="line-2"></div>
+					<div class="line-3"></div>
+				</div>
+
+				<ul class="navbar-ul hide-burger-menu burger-menu-dropdown">
+					<li><a href="/">Home</a></li>
+					<li><a href="/register">Register</a></li>
+					<li><a href="/rules">Rules</a></li>
+					<li>
+						<a
+							href="https://ourtech.community/coc"
+							target="_blank"
+							rel="noreferrer"
+							>Code of Conduct</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://t.me/OurTechComm"
+							target="_blank"
+							rel="noreferrer"
+							>Join us on Telegram</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/two-years"
+							target="_blank"
+							rel="noreferrer"
+							>Two Years of OTC</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/team"
+							target="_blank"
+							rel="noreferrer"
+							>Our Team</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://links.ourtech.community"
+							target="_blank"
+							rel="noreferrer"
+							>All Links</a
+						>
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks Rules</h1>
+
+		<main id="main-content">
+			<ul>
+				<li>The mode of OTC Talks is Online.</li>
+				<li>
+					Duration of the lightning talk: 15 - 25 minutes from 10:45
+					PM IST on a Saturday, followed by the CatchUp on the
+					respective day.
+				</li>
+				<li>QnA/Discussion after the talk: About 15 minutes.</li>
+				<li>
+					The Talks are open for all and can be attended by anyone.
+				</li>
+				<li>
+					IMPORTANT:
+					<ul>
+						<li>
+							Only topics from Tech and related fields (Eg: mental
+							health, salary negotiations, etc) are permitted.
+						</li>
+						<li>
+							We highly encourage demonstrations over
+							presentations, but speakers are free to do as they
+							please.
+						</li>
+						<li>
+							Promotional or sponsored content is not permitted.
+						</li>
+					</ul>
+				</li>
+				<li>
+					Please follow the
+					<a
+						href="https://ourtech.community/coc"
+						target="_blank"
+						rel="noreferrer"
+						>Code of Conduct</a
+					>
+					while attending our talks. Any violations can be reported to
+					<a
+						href="https://ourtech.community/team"
+						target="_blank"
+						rel="noreferrer"
+						>the organisers</a
+					>
+					or on
+					<a href="mailto:contact@ourtech.community"

```

Comment:
This `a` tag should have a `rel` attribute with the value `noreferrer`.

Reply:
https://github.com/OurTechCommunity/meetup/blob/a8d7ea90c0ac5bc455d9576e27590d18857e72e7/rules.html#L144-L146

*coughs*

Reply:
Fixed this though.

Reply:
Yeah the e-mail in the OTC MeetUp rules should be fixed. Anyways, thank you for fixing this.